service/service_unit_test.go

```diff
@@ -105,6 +105,22 @@ func TestGetVolSize(t *testing.T) {
 			},
 			sizeKiB: 0,
 		},
+		{
+			// Requesting a size with decimal part that rounds up to next multiple of 8 GiB
+			cr: &csi.CapacityRange{
```

Comment:
Could we add more test cases with higher decimal precision values? Additionally, let's use the same input that helped us identify this issue for consistency.

Reply:
Done